### DISCUSSION TITLE
how to disable discount summary fileds in sales order module version 6.1.0

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#15059] [How to]
![Capture](https://user-images.githubusercontent.com/75428743/128988663-c60721c2-e110-46b8-90e8-3324f17546f8.PNG)

**Comment by Akaki-1991** [comment]
....

**Reply by vovpff** [reply]
I'm not sure but try to remove discount and tax columns for advanced fields?

**Reply by Akaki-1991** [reply]
hello, can't remove. there is no fields

**Reply by rskrzypczak** [reply]
What vovpff wrote is correct.
If you do not have these fields, and you are getting blocks, you must have messed up something in the code yourself, which you have to verify yourself.

https://github.com/YetiForceCompany/YetiForceCRM/blob/9d0f238b114cbdadf9265c4415183e0ca18fcef9/layouts/basic/modules/Vtiger/Edit/InventorySummary.tpl#L5

**Reply by vovpff** [reply]
@Akaki29  I think you need to remove this fields in advanced block
![image](https://user-images.githubusercontent.com/8545791/129524418-2f2ddcc9-d15a-4d7e-ae92-3d90545df109.png)
